Tests/IntegrationTests/start-system-under-test-docker.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+set -e
+
+ARCH=$(uname -m)
+if [ "$ARCH" = "arm64" ]; then
+  export DB_IMAGE=arm64v8/mysql:8
+elif [ "$ARCH" = "x86_64" ]; then
+  export DB_IMAGE=mysql:8
+else
+  echo "Unknown architecture"
+  exit 1
+fi
+
+function dc() {
+    if [ -n "$(docker compose version)" ]; then
+        # use the docker composer plugin
+        docker compose -f ./Tests/IntegrationTests/docker-compose.system-under-test.yaml $@
+    else
+        # legacy docker compose standalone
+        docker-compose -f ./Tests/IntegrationTests/docker-compose.system-under-test.yaml $@
+    fi
+}
+
+echo "#############################################################################"
+echo "# Start docker environment...                                               #"
+echo "#############################################################################"
+dc down
+dc up -d
+dc exec -T php bash <<-'BASH'
+    rm -rf /usr/src/app/*
+BASH
+docker cp "$(pwd)"/Tests/IntegrationTests/. "$(dc ps -q php)":/usr/src/app
```

Comment:
Todo: Lets use paths relative to the shellscript?

Reply:
Fine by me, I didn't want to put too much time into it and copy pasted the most stuff.

It's clear, that we could build a better system for all of this but I needed a faster feedback loop fast 😬 
I want to build tests for Bugs I fix and wasting 2h just for building the test and another 2 for running them to see if the fix works and we don't mess up something else ist just too much ^^